tests/validation-target.test.js

```diff
@@ -164,12 +167,10 @@ await test("'target' configuration file option", async (t) => {
       ["tstyche.config.json"]: JSON.stringify(config, null, 2),
     });
 
-    const { exitCode, stderr, stdout } = await spawnTyche(fixtureUrl, [], {
+    const { exitCode, stderr } = await spawnTyche(fixtureUrl, [], {
       env: { ["TSTYCHE_TYPESCRIPT_PATH"]: "" },
     });
 
-    assert.match(stdout, /^adds TypeScript/);
-
     const expected = [
       "Error: Cannot use 'current' as a target. Failed to resolve the path to the currently installed TypeScript module.",
```

Comment:
This is the problem being solved. Basically TypeScript is not installed, but TSTyche installs it just to tell that is not installed. Hm..